lib/input_validator.rb

```diff
@@ -0,0 +1,17 @@
+module TicTacToe 
+  class InputValidator
+
+    def is_size_valid?(size)
+      size <= 10 && size > 2
+    end
+
+    def is_marker_valid?(marker)
+      marker.capitalize != "X" && marker.length == 1 ? true : false
+    end
+
+    def is_move_valid?(move, size)
+      cell_nums = size * size
+      move <= cell_nums && move > 0 ? true : false
```

Comment:
See previous note ^^ Logic like this - `move <= cell_nums && move > 0` - requires more than 5 seconds of mental calculation and thus should be split out into a well-named private method.